cmd/migrate/migrate.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/ledgerwatch/bolt"
+	"github.com/ledgerwatch/turbo-geth/common/dbutils"
+)
+
+func encodeToCBOR(buf []byte) ([]byte, error) {
+	acc, err := decodeRLP(buf)
+
+	if err != nil {
+		return nil, err
+	}
+
+	res := make([]byte, acc.EncodingLengthForStorage())
+	acc.EncodeForStorage(res)
+	return res, nil
+}
+
+func convertDatabaseToCBOR(db *bolt.DB) error {
+	return db.Update(func(readTx *bolt.Tx) error {
```

Comment:
You probably want to split it in multiple transactions, otherwise it will run out of memory. I suggest limiting 1 transaction to 100k `Put` operations